tests/run.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 cd testing
-cp ./docs/management_api.yml .
+sleep 5
```

Comment:
can you try with something like this? https://github.com/mendersoftware/deviceauth/blob/master/tests/run.sh#L11